### ISSUE TITLE
Fix Sugested for running client in Linux

### ISSUE THREAD

**Issue by cmcglownjr** [FAForever/client#1129]
Hi FAF developers,

First I'd like to apologize if this is done wrong but I'm pretty green at raising issues with open-source projects. I was trying to build and run this client in my linux system, Kubuntu 20.04.3 but I kept running into an authentication problem while logging in. The problem was it was looking for the "faf-uid.exe" and that wasn't available since I use linux and not windows. I put the "faf-uid" file for linux in the "lib" directory but it still didn't work.

I decided to look through the source code to find the problem. Under "src/util/__init__.py" there was an error on line 473 where it detects the platform and assigns a path to the variable "exe_path". If the os is "win32" the the variable is set to: 
```
os.path.join(fafpath.get_libdir(), "faf-uid.exe")
```
but the else clause just has:
```
exe_path = "faf-uid"
```
I changed the else clause to:
```
exe_path = os.path.join(fafpath.get_libdir(), "faf-uid")
```
and the client was able to log me in. Please make this correction to your code to prevent this error from happening to any future users. Also consider creating an AppImage to distribute this software easily.

Thank you and I apologize again for not understanding how to properly submit bug reports and fixes on Github.

**Comment by GeorgeZack** [comment]
Do you happen to have a fork for this change?

**Comment by cmcglownjr** [comment]
No I didn't create a fork. If you need me to then I can create the fork, apply the change, and submit a pull request. I don't work in collaborative projects often so I was hesitant with getting involved. Also it seems like the FAF Client is now using Java instead of Python.

**Comment by GeorgeZack** [comment]
Nope, I'm the same I don't really fork things often. The Java client is giving me a hard time on Artix using DWM so I was hoping the Python client would work. I ended up getting it to work but still had issues. Thanks for responding

**Comment by andrewtvuong** [comment]
Is there a lead handling all this? New to opensource as well, but want to contribute. Was interested to write some Python but like others have mentioned Java seems to be launching the game for me now and readme seems to say Python is legacy now. I wonder how it is still being used, if at all?

Thanks!

**Comment by Brutus5000** [comment]
This client has ~10 active users and only 1 semi-active developer that kept it alive (@Gatsik). I'm not sure if he is looking into the issues here and/or interested in collaboration with others.
